db/migrate/20170710132935_custom_button_to_classes.rb

```diff
@@ -0,0 +1,144 @@
+class CustomButtonToClasses < ActiveRecord::Migration[5.0]
```

Comment:
To @lpichler's point, almost all of the migrations start with a verb, so `ConvertCustomButtonToClasses`?